web/wp-content/themes/dctx/template-no-title.php

```diff
@@ -0,0 +1,29 @@
+<?php 
+
+/**
+* Template Name: No Title Page
+*
+* @package WordPress
+* @subpackage dctx
+* @since dctx 1.0
+*/
+	
```

Comment:
:no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).